File: 1252-break-a-palindrome/1252-break-a-palindrome.py
class Solution:
    def breakPalindrome(self, p: str) -> str:
        res = list(p)
        n = len(p)
        if n ==1 : return ""
        print(res)
        for i in range(n//2):
            if res[i] != 'a':
                res[i] = 'a'
                return ''.join(res)
        
        res[-1] = 'b'
        return ''.join(res)

File: 1252-break-a-palindrome/1252-break-a-palindrome.go
func breakPalindrome(p string) string {
    runes := []rune(p)
    n := len(runes)
    if n == 1 {
        return ""
    }

    for i := 0; i < n/2; i++ {
        if runes[i] != 'a' {
            runes[i] = 'a'
            return string(runes)
        }
    }

    runes[n-1] = 'b'
    return string(runes)
}